src/components/Nav/Nav.js

```diff
@@ -1,7 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import UserForm from './UserFormComponent/userForm';
+import { userFormInfo } from './NavData';
+
```

Comment:
```suggestion
import { userFormInfo } from './NavData';
const [LOGIN_INFO, SIGNUP_INFO] = userFormInfo

```
이렇게 사용하면 인덱스 사용하지 않아도 되고요 그런데 꼭 하나로 합쳐놓기 보다는 저는 그냥 애초에 각각의 변수로 export 할거 같아요

Reply:
> 이렇게 사용하면 인덱스 사용하지 않아도 되고요 그런데 꼭 하나로 합쳐놓기 보다는 저는 그냥 애초에 각각의 변수로 export 할거 같아

하나로 합쳐야한다는 생각만 했었네요! 감사합니다 !!